tests/vmi_lifecycle_test.go

```diff
@@ -1006,6 +1006,57 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 					fmt.Sprintf("Expected CPU model to equal to the default (%v)", v1.DefaultCPUModel),
 				)
 			})
+			It("[sig-compute]should add node selector to virt-launcher when setting default cpuModel in kubevirtCR", func() {
+				if len(supportedCpuModels) < 1 {
+					Skip("Must have at least one supported cpuModel for this test")
+				}
+				defaultCPUModel := supportedCpuModels[0]
+				config := originalConfig.DeepCopy()
+				config.CPUModel = defaultCPUModel
+				tests.UpdateKubeVirtConfigValueAndWait(*config)
+
+				vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
+				_, err = virtClient.VirtualMachineInstance(vmi.Namespace).Create(vmi)
+				Expect(err).ToNot(HaveOccurred(), "Should create VMI")
+				tests.WaitForSuccessfulVMIStart(vmi)
+				By("Fetching virt-launcher pod")
+				virtLauncherPod := tests.GetRunningPodByVirtualMachineInstance(vmi, vmi.Namespace)
+				foundCpuModelSelector := false
+				for key, _ := range virtLauncherPod.Spec.NodeSelector {
+					if strings.Contains(key, defaultCPUModel) {
+						foundCpuModelSelector = true
+					}
+				}
+				Expect(foundCpuModelSelector).To(BeTrue())
```

Comment:
nit: would add description here (or else the error would look like `Expected false to be true`)

Reply:
Done.
Thanks!

Reply:
right, fixed it as well 
Thanks1!